src/app/Services/UserService.php

```diff
@@ -75,4 +76,16 @@ public function resetPassword($token, $newPassword): User
 
         return $user;
     }
+
+    /**
+     * Get User Profile
+     */
+
+    public function getUserProfile(int $id): array
+    {
+        $user = User::with('profile')->findOrFail($id);
```

Comment:
hmm I noticed youre not using try catch, What if this logic fails?

Reply:
Added try catch